modules/aws-emr-cluster/outputs.tf

```diff
@@ -17,9 +17,23 @@ output "release_label" {
   description = "The release label for the Amazon EMR release."
 }
 
-output "core_group_instance_count" {
-  value       = var.core_group_instance_count
```

Comment:
Good that you added these new outputs.

Could we keep `core_group_instance_count` output. 

Maybe we could do something like:
```
value = var.core_instance_on_demand_count +  var.core_instance_spot_count
```

The total is likely going to be the interesting value here so that other modules (if I had to guess how this value is used).